prod-aws/kafka-shared-msk/customer-billing/customer-billing.tf

```diff
@@ -0,0 +1,13 @@
+resource "kafka_topic" "mail_sender_deadletter" {
+  name               = "bex.internal.mail_sender_deadletter"
+  replication_factor = 3
```

Comment:
Are you mirroring this topic or is it a new topic you are creating? Wondering if you need 10 partitions for a deadletter topic.

Reply:
I is a new topic. Good point, I reduced the number of partition to 1.